oat-sa/tao-core
# Fix/edge checkboxes

https://oat-sa.atlassian.net/browse/TAO-1585
https://oat-sa.atlassian.net/browse/TAO-1586
Background of checkboxes/radio buttons were badly positioned

Test instructions
- create an item with two choice interactions, one with radio buttons, one with checkboxes
- view the item in MS Edge
- as a helper you can set the background of the interactions in the browser inspector to a color other than white

Expected behaviour:
- the checkboxes/radio buttons should have a white background

Note: The screen shot was taken with TAO-3273 already in place, hence the second column - you can safely ignore that.

![2016-10-28 10_52_40-tao 3 2 0-sprint37 and 2 more pages - microsoft edge](https://cloud.githubusercontent.com/assets/8349692/19800802/bebd5a0c-9cfc-11e6-8afb-78d9fa265177.png)

I'm on Windows and can share a screen or lent my Laptop for validation
